src/sentry/api/endpoints/organization_sdk_updates.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+from datetime import datetime, timedelta
+from sentry.api.bases.organization import OrganizationEndpoint
+
+from itertools import groupby
+from rest_framework.response import Response
+from sentry.sdk_updates import SdkIndexState, SdkSetupState, get_suggested_updates
+
+from sentry.utils.compat import map
+from sentry.snuba import discover
+
+
+def by_sdk_name(sdk):
+    return sdk["sdk.name"]
+
+
+def by_project_id(sdk):
+    return sdk["project.id"]
+
+
+class OrganizationSdkUpdates(OrganizationEndpoint):
+    def get(self, request, organization):
+
+        project_ids = self.get_requested_project_ids(request)
+        projects = self.get_projects(request, organization, project_ids)
+
+        result = discover.query(
+            query="has:sdk.version",
+            selected_columns=["project", "sdk.name", "sdk.version", "last_seen()"],
+            orderby="-project",
+            params={
+                "start": datetime.now() - timedelta(days=1),
+                "end": datetime.now(),
+                "organization_id": organization.id,
+                "project_id": [p.id for p in projects],
+            },
+            referrer="api.organization-sdk-updates",
+        )
+
+        # Build datastructure of the latest version of each SDK in use for each
+        # project we have events for.
+        sdks_by_project = [
+            (
+                project_id,
+                [
+                    {
+                        "name": sdk_name,
+                        "version": sorted(
+                            sdks, key=lambda v: [int(u) for u in v["sdk.version"].split(".")]
```

Comment:
Can't versions have stuff like `alpha` etc. in the version text? eg.
https://github.com/getsentry/sentry-python/releases/tag/0.1.0-preview3 Should we just skip releases that aren't in the format we expect?

Reply:
Yeah this would likely break on that, you're right.